WorkloadManagementSystem/Service/WMSAdministratorHandler.py

```diff
@@ -63,8 +72,11 @@ def export_getSiteMask(cls, siteState='Active'):
   @classmethod
   def export_getSiteMaskStatus(cls, sites=None):
     """ Get the site mask of given site(s) with columns 'site' and 'status' only
-    """
 
+        :param list or basestring sites: sites in list or site
```

Comment:
the "X or Y" doesn't work in `:param`, this only works with `:type parName: X or Y`.